exmg-paper-combobox-icons.html

```diff
@@ -0,0 +1,10 @@
+<link rel="import" href="../iron-iconset-svg/iron-iconset-svg.html">
+
+<iron-iconset-svg name="exmg-paper-combobox-icons" size="24">
+<svg>
+  <defs>
+    <g id="arrow-drop-down"><path d="M7 10l5 5 5-5z"></path></g>
+    <g id="clear"><path d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/></g>
```

Comment:
clear icon can be removed from set

Reply:
It will conflict with the delete button on the combobox, and I would like to keep it as close to the token input as possible.